lms/djangoapps/edxnotes/tests.py

```diff
@@ -1009,22 +1009,6 @@ def test_edxnotes_view_is_disabled(self):
         self.assertEqual(response.status_code, 404)
 
     @patch.dict("django.conf.settings.FEATURES", {"ENABLE_EDXNOTES": True})
-    @ddt.unpack
-    @ddt.data(
-        {'side_effect': EdxNotesServiceUnavailable},
-        {'side_effect': EdxNotesServiceUnavailable},
-    )
-    def test_edxnotes_view_500_error(self, side_effect):
-        """
-        Tests that 500 status code is received for EdxNotesServiceUnavailable or EdxNotesServiceUnavailable exceptions.
-        """
-        with patch("edxnotes.views.get_notes", autospec=True) as mock_get_notes:
-            mock_get_notes.side_effect = side_effect
-            enable_edxnotes_for_the_course(self.course, self.user.id)
-            response = self.client.get(self.notes_page_url)
-            self.assertEqual(response.status_code, 500)
```

Comment:
As discussed there is no straightforward way to test this and so we can skip it and we can assume that if a exception is raised by the view, it will be appropriately handled by Django.